tests/copy_assigner_outputs.sh

```diff
@@ -1,5 +1,5 @@
-SRC_DIR="build/examples/cpp"
-DEST_DIR="$1"
+SRC_DIR="$1"
+DEST_DIR="$2"
```

Comment:
Can't find actual change of usage with two arguments instead of one.

Reply:
Reverted